README.md

```diff
@@ -5,14 +5,56 @@ Python Machine Learning development in a breeze.
 
 ![Arcus](https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png)
 
+# Positioning
+
+With Arcus we are offering an open source library that streamlines ML development with Python and offers out of the box implementations for common scenarios.
+
+
+# Features
+
+Coming soon
+
 # Installation
 
+The Arcus packages are available through PyPi and can be installed through pip, by executing the following command:
+
 ```shell
 PM > pip3 install arcus-ml
 ```
 
-# Documentation
-All documentation can be found on [here](https://machine-learning.arcus-azure.net/).
+Upgrading to the latest version can be done by executing the following pip command:
+
+```shell
+PM > pip3 install --upgrade arcus-ml 
+```
+
+Every CI build pushes a dev package to the PyPi feed.  And when committed, an alpha release is been published on the same feed.  These packages can be installed or up  grade, by leveraging the `--pre` argument for `pip`.
+
+```shell
+PM > pip3 install --upgrade --pre arcus-ml
+```
+
+# Local development
+    
+It can be quite common that you are using the arcus-ml or arcus-azureml packages on other projects and you need some changes or additional functionality being added to the package.  Obviously, it's possible to follow the entire release pipeline (make a PR, get it approved and merged, wait for package to appear on the PyPi feed and upgrade the package).  This workflow is too tedious and will not increase your productivity.
+
+The approach to go, is to leverage the following command, which will add a symbolic link to your development directory from the python packages.  That way, you always refer to the latest code that is on your development environment.  It is advised, though not required to leverage virtual environment (like with conda) for this.
+
+```shell
+pip install -e /path-to-arcus
+```
```

Comment:
Already a good first step to having some docs on local development. I would love to see some more practical information.
* Link to PyPi feed
* Link to how to set such a virtual environment

Reply:
Created #98 for this.  Focus of this PR should be on the actual build + doc generation.